src/integrationTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/SensitivityTest.java

```diff
@@ -0,0 +1,213 @@
+import uk.gov.hmcts.reform.pip.account.management.model.CreationEnum;
+import uk.gov.hmcts.reform.pip.account.management.model.ListType;
+import uk.gov.hmcts.reform.pip.account.management.model.PiUser;
+import uk.gov.hmcts.reform.pip.account.management.model.Roles;
+import uk.gov.hmcts.reform.pip.account.management.model.Sensitivity;
+import uk.gov.hmcts.reform.pip.account.management.model.UserProvenances;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(classes = {AzureConfigurationClientTest.class, Application.class},
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+@ActiveProfiles(profiles = "test")
+@AutoConfigureEmbeddedDatabase(type = AutoConfigureEmbeddedDatabase.DatabaseType.POSTGRES)
+@WithMockUser(username = "admin", authorities = { "APPROLE_api.request.admin" })
+class SensitivityTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    private static final String ROOT_URL = "/account";
+    private static final String PI_URL = ROOT_URL + "/add/pi";
+    private static final String ISSUER_EMAIL = "issuer@email.com";
+    private static final String ISSUER_HEADER = "x-issuer-email";
+    private static final String EMAIL = "a@b.com";
+    private static final String URL_FORMAT = "%s/isAuthorised/%s/%s/%s";
+
+    private static final String TRUE = "true";
+    private static final String FALSE = "false";
+
+    private static final String TRUE_MESSAGE = "Should return true";
+    private static final String FALSE_MESSAGE = "Should return false";
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private String createUserAndGetId(PiUser validUser) throws Exception {
+        MockHttpServletRequestBuilder setupRequest = MockMvcRequestBuilders
+            .post(PI_URL)
+            .content(objectMapper.writeValueAsString(List.of(validUser)))
+            .header(ISSUER_HEADER, ISSUER_EMAIL)
+            .contentType(MediaType.APPLICATION_JSON);
+
+        MvcResult userResponse = mockMvc.perform(setupRequest).andExpect(status().isCreated()).andReturn();
+        ConcurrentHashMap<CreationEnum, List<Object>> mappedResponse =
+            objectMapper.readValue(userResponse.getResponse().getContentAsString(),
+                                   new TypeReference<>() {});
+        return mappedResponse.get(CreationEnum.CREATED_ACCOUNTS).get(0).toString();
+    }
+
+    @Test
+    void testIsUserAuthenticatedReturnsTrueWhenPublicListAndVerified() throws Exception {
+        PiUser user = new PiUser();
+        user.setEmail(EMAIL);
+        user.setProvenanceUserId(UUID.randomUUID().toString());
+        user.setUserProvenance(UserProvenances.PI_AAD);
+        user.setRoles(Roles.VERIFIED);
+
+        String createdUserId = createUserAndGetId(user);
```

Comment:
repeated in every test, feels like this should be moved to a beforeEach

Reply:
The user is slightly different each time so I've had to keep this line here, however I've moved some of the duplicated bits above that sets up the user into the beforeEach

Reply:
ok sweet!